src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Job2dDriverDrawingInterpreter.java

```diff
@@ -8,11 +8,13 @@
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
-	private int startX = 0, startY = 0;
+public class Job2dDriverDrawingInterpreter extends DrawPanelController implements Job2dDriver {
```

Comment:
Why Interpreter?
Inheritance is not used (no super drawLine calls). Should be removed!